server/config.json

```diff
@@ -19,8 +19,8 @@
       "limit": "100kb"
     },
     "cors": false,
-    "errorHandler": {
-      "disableStackTrace": false
+    "strong-error-handler": {
+      "isDebugMode": false
```

Comment:
this is for strong-remoting, I think keeping it as `errorHandler` might be easiest for users 
https://github.com/strongloop/strong-remoting/blob/b03dfe8ec53d4ca694a2e438d6b5dadff0824305/lib/rest-adapter.js#L316
```js
"errorHandler" : {
  "debug": true,
  "log": false
}
```